ops/ansible/playbooks-ccs/launch_bfd-server.yml

```diff
@@ -6,6 +6,7 @@
 - name: Launch BFD Data Server
   hosts: localhost
   gather_facts: yes
+  gather_subset: '!all,!min,facter'
```

Comment:
Here I attempted to further cut down the launch time by optimizing ansible fact gathering. It didn't have too much of an effect, likely due to the fact that we require `facter` facts since we pull from it in this statement below: `ansible_facts['facter_ec2_metadata']['placement']['availability-zone']`

Facter seems to be expensive, so perhaps this could be replaced with a custom ansible role that performs a AWS EC2 metadata lookup?